lib/cuffsert/presenters.rb

```diff
@@ -76,6 +77,16 @@ def on_event(event)
       end
     end
 
+    def on_error(err)
+      case err
+      when CuffSertError
+        @renderer.abort(err)
+      else
+        super(err)
+      end
+      exit(1)
```

Comment:
In general I don't like calling exit outside "the main function" as it means that the feature composes very badly (and makes it somewhat more difficult to write tests), but maybe it works as intended in this case.

Reply:
I agree. However, Cuffsert uses the "official" RxRuby which has no "await" interface, so the alternative would be to use a mutex or some similar construct to block in `#main` until the presenter is finished. Since there is no indication that RxRuby is maintained, this would have to be a local solution. My experience with `Mutex`, `Queue` and `ConditionalVariable` is somewhat mixed, so in the end I opted for this ugliness.